src/lib/Libdb/db_postgres_resv.c

```diff
@@ -84,29 +83,28 @@ pg_db_prepare_resv_sqls(pbs_db_conn_t *conn)
 		") "
 		"values "
 		"($1, $2, $3, $4, $5, $6, $7, $8, $9, $10, $11, $12, $13, "
-		"$14, $15, localtimestamp, localtimestamp, hstore($16::text[]))");
-	if (pg_prepare_stmt(conn, STMT_INSERT_RESV, conn->conn_sql, 16) != 0)
+		"$14, localtimestamp, localtimestamp, hstore($15::text[]))");
+	if (pg_prepare_stmt(conn, STMT_INSERT_RESV, conn->conn_sql, 15) != 0)
 		return -1;
 
 	snprintf(conn->conn_sql, MAX_SQL_LENGTH, "update pbs.resv set "
 		"ri_queue = $2, "
 		"ri_state = $3, "
 		"ri_substate = $4, "
-		"ri_type = $5, "
-		"ri_stime = $6, "
-		"ri_etime = $7, "
-		"ri_duration = $8, "
-		"ri_tactive = $9, "
-		"ri_svrflags = $10, "
-		"ri_numattr = $11, "
-		"ri_resvTag = $12, "
-		"ri_un_type = $13, "
-		"ri_fromsock = $14, "
-		"ri_fromaddr = $15, "
+		"ri_stime = $5, "
```

Comment:
How will this affect upgrades?  This affects the schema, so when the schema changes, will the database as well?

Reply:
I hadn't thought about that. You are correct, this changes the schema, I'm not sure how upgrades will work. Do you know how we usually handle this?

Reply:
I think habitat updates the schema and the tables in the database.  I'm not exactly sure how it is done.  I suspect @subhasisb will know.